modules/swagger-codegen/src/test/resources/2_0/petstore.json

```diff
@@ -880,7 +880,8 @@
       "properties": {
         "id": {
           "type": "integer",
-          "format": "int64"
+          "format": "int64",
+          "x-unique-id-key"  : true
```

Comment:
Let's keep the petstore.json/yaml as it's and add additional test-related model properties in `modules/swagger-codegen/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing.yaml` instead (you will need to update `./bin/objc-petstore.sh` to use petstore testing yaml file)